### PR TITLE
V22 week2 fixes

### PR DESCRIPTION
**Pull request by jakobsj**
I updated notebooks 1, 2, 3 and 4 with year and syntax changes. I believe @gfardell already updated notebook 5 (laminography) elsewhere.  This PR may need to be pointed to a different branch, v22_all or something?

### CODE REVIEW

File: examples/2_Iterative/02_tikhonov_block_framework.ipynb [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "#   See the License for the specific language governing permissions and\n",
     "#   limitations under the License.\n",
     "\n",
-    "#   Copyright 2019 UKRI-STFC, The University of Manchester, Technical University of Denmark\n",
+    "#   Copyright 2019-2022 UKRI-STFC, The University of Manchester, Technical University of Denmark\n",
```
Comment (gfardell) [inline examples/2_Iterative/02_tikhonov_block_framework.ipynb]: I think this needs the new header, not just the date range.

File: examples/2_Iterative/03_PDHG.ipynb [hunk 2]
```diff
@@ -718,8 +732,8 @@
     "plt.figure(figsize=(30,8))\n",
     "plt.rcParams.update({'font.size': 15})\n",
     "plt.rcParams.update({'lines.linewidth': 5})\n",
-    "plt.plot(fbp_recon.subset(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'FBP')\n",
-    "plt.plot(pdhg_l1.solution.subset(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'L1 regularisation')\n",
+    "plt.plot(fbp_recon.get_slice(horizontal_y = int(ig2D.voxel_num_y/2)).as_array(), label = 'FBP')\n",
```
Comment (gfardell) [inline examples/2_Iterative/03_PDHG.ipynb:685]: Can we use assign the row number to a variable as it's used many times:
`row_num = int(ig2D.voxel_num_y/2)`

plt.plot(fbp_recon.get_slice(horizontal_y =row_num).as_array(), label = 'FBP')\n

File: examples/2_Iterative/04_SPDHG.ipynb [hunk 3]
```diff
@@ -23,7 +23,7 @@
     "#   See the License for the specific language governing permissions and\n",
     "#   limitations under the License.\n",
     "\n",
-    "#   Copyright 2019 UKRI-STFC, University of Bath\n",
+    "#   Copyright 2019-2022 UKRI-STFC, University of Bath\n",
```
Comment (gfardell) [inline examples/2_Iterative/04_SPDHG.ipynb]: header needs updated text

File: examples/2_Iterative/04_SPDHG.ipynb [hunk 4]
```diff
@@ -153,7 +153,11 @@
    "source": [
     "Now, we would like to setup and run SPDHG algorithm for the above minimisation problem. \n",
     "\n",
-    "First, we'll load the **walnut** dataset found in [Jørgensen_et_al](https://zenodo.org/record/4822516#.YLXyAJMzZp8) and work on the central slice only in the following demonstration. See more details in the PDHG notebook."
+    "First, we'll load the **walnut** dataset found in [Jørgensen_et_al](https://zenodo.org/record/4822516#.YLXyAJMzZp8) and work on the central slice only in the following demonstration. See more details in the PDHG notebook.\n",
+    "\n",
+    "The data set can be downloaded here, after which you will need to unzip the file and update `pathname` below:\n",
+    "\n",
+    "https://zenodo.org/record/4822516/files/walnut.zip?download=1"
```
Comment (gfardell) [inline examples/2_Iterative/04_SPDHG.ipynb]: You can remove '?download=1'

**Review by gfardell — APPROVED** [review]
It's really hard to check these thoroughly through github. I would suggest we spend an hour together to go through them all (5mins per notebook) so we can keep a level of consistency.

**Review by lauramurgatroyd — APPROVED** [review]

**Pull request merged**